src/utils/dom.ts

```diff
@@ -159,10 +159,11 @@ export async function focusElement(el: CalciteFocusableElement): Promise<void> {
 interface GetSlottedOptions {
   all?: boolean;
   direct?: boolean;
+  matches?: string;
   selector?: string;
 }
 
-const defaultSlotSelector = "> :not([slot])";
+const defaultSlotSelector = ":not([slot])";
```

Comment:
I'm not sure why I had the `> ` but this might be something we need??

Reply:
Ok, so can't do direct selector because of the scope. Instead, have to filter child nodes first.

I tried using `:scope > ` but that wasn't working either.
https://developer.mozilla.org/en-US/docs/Web/CSS/:scope

Reply:
Interesting. Do the current spec tests cover this? Wonder why it didn't pick this up.

Reply:
> Do the current spec tests cover this? Wonder why it didn't pick this up.

We weren't using a selector like this for all default slot queries.